src/sources/logstash.rs

```diff
@@ -51,13 +51,13 @@ pub struct LogstashConfig {
 
     /// The size of the receive buffer used for each connection.
     ///
-    /// This generally should not need to be changed.
+    /// Generally, this should not need to be changed.
```

Comment:
When would it need to be changed?

Reply:
Same comment applies to the below ones 😄

Reply:
Ah, I missed that you'd already commented to this effect Jesse in my below reply to one of the other places it was flagged.

Reply:
Thanks @jszwedko @neuronull for explaining! You both seem to agree that we could drop the "Generally, this should not need to be changed" statement, so I'll go ahead and remove all instances of where I've flagged it.

Reply:
Removed all